classifiers/reference_complexity/maximum_sentence_complexity/config.py

```diff
@@ -0,0 +1,61 @@
+from util.configs import build_classifier_function_config
+from util.enums import State, RefineryDataType, BricksVariableType, SelectionType
+from . import maximum_sentence_complexity, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_classifier_function_config(
+        function=maximum_sentence_complexity,
+        input_example=INPUT_EXAMPLE,
+        issue_id=378,
+        tabler_icon="ThreeDCubeSphere",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        type="python_function",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "reference_complexity",
+            "text_analysis"
+        ],  # first entry should be parent directory
```

Comment:
Also `question_complexity`

Reply:
Yes, I have added it. Although, as there is no actual group question_complexity containing bricks, it won't actually show the grouping in the frontend. It we just need this information from the backend, this is fine though.